### ISSUE TITLE
It takes an abnormally slow amount of time to activate a script in the Cheat Engine table that contains the AOBScan command

### ISSUE THREAD

**Issue by luofeng3000** [cheat-engine/cheat-engine#2560]
I've been using Cheat Engine for a while now, and at some point, every script in the CT cheat table that contains the AOBScan command takes over ten seconds to activate after clicking the activation box. During this time, Cheat Engine displays "Not Responding" in the software's title bar.

If I manually do the work that AOBScan does in the script ,manually search for the byte array, it's almost instantaneous, and there is no lag.

It's worth noting that this is not because my computer's performance is too weak or because the game is too large. I even bought a new desktop with an i7 12th generation processor, but the problem persists.

I searched online, and it's not an isolated issue; another user has experienced the same problem. I've included the link below.

[[Suddenly Slow AOBScan on previously working tables](https://fearlessrevolution.com/viewtopic.php?t=19471)]

Do you have any thoughts on how to solve this issue?

**Comment by cheat-engine** [comment]
Did you also reinstall windows with a new clean user?

Does deactivating a record and then reactivate take just as long?

**Comment by luofeng3000** [comment]
1.Yes,new PC and new system.
2.No,deactivating a record is very fast, reactivating  takes 12-13 second same as first time.  Scripts that do not contain the aobscan command are quick to activate.
3.No debugger attached.

**Comment by luofeng3000** [comment]
Yes, I checked the settings. They weren't checked anyway. Array of Byte Scan is fast , AOBScan script is slow.
My situation is after I found problems on my old computer, I switched to a new computer with a new operating system and new installed CE. The problem still reappears. I don’t think this is a setting issue.
